superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilters.test.ts

```diff
@@ -202,6 +202,7 @@ function openVerticalFilterBar() {
 
 function setFilterBarOrientation(orientation: 'vertical' | 'horizontal') {
   cy.getBySel('filterbar-orientation-icon').click();
+  cy.wait(250);
```

Comment:
Can we add timeout to `getBySel` instead of this `wait`? Like `cy.getBySel('dropdown-selectable-info', { timeout: 250 })`

Reply:
Actually, not sure which method do we prefer in Superset, so feel free to leave it as is 🙂

Reply:
Nevermind, I didn't notice that it's merged 😄

Reply:
Haha no worries!  For this one, the timeout was for after the `.click()` to open the menu.  The issue seemed to be that the loading of the page caused some scrolling action while the menu was opening, which disrupted Cypress's ability to click on the menu item, so this `.wait` hopefully will let the scrolling finish in more cases.